builtin/providers/kubernetes/provider.go

```diff
@@ -0,0 +1,171 @@
+package kubernetes
+
+import (
+	"bytes"
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/hashicorp/terraform/terraform"
+	"github.com/mitchellh/go-homedir"
+	kubernetes "k8s.io/kubernetes/pkg/client/clientset_generated/release_1_5"
```

Comment:
I really wanted to use that library when starting the work on K8S provider, but it was practically impossible to use it as a vendored dependency due to some upstream issues/conflicts which I described in https://github.com/hashicorp/terraform/pull/12230#issue-210035666

We might switch to that library at some point in the future to reduce the number of vendored LOC and because that's the library that should be used for that purpose, but the main goal is to get initial coverage of all stable (v1) k8s resources for now so re-vendoring is not currently on my radar.

I hope that's understandable 😃